packages/nimble-components/src/table/specs/README.md

```diff
@@ -1,78 +1,276 @@
+
+-   The element that will serve as the scollable element
+-   An estimated height for each row
+-   The total count of rows in the data
+
+With this set of information, the Nimble Table will be able to register a callback to the TanStack Virtual `onChange` which will happen any time the scrollable element scrolls. In that handler the Nimble Table can retrieve the set of virtual items from TanStack Virtual (i.e. `getVirtualItems()`), which represent the total set of rows that should be displayed, and contain the state information that allows the Nimble Table to retrieve the appropriate data from the TanStack Table model to apply to each rendered row, as well as the position each row should be rendered.
+
+_Placeholder for other implementation details_
+
+### States
+
+Placeholder
+
+### Accessibility
+
+The `nimble-table` should align either to the W3C [grid interaction model](https://w3c.github.io/aria-practices/#grid) or the [table interaction model](https://w3c.github.io/aria-practices/#table) (TBD).
 
-The bulk of the effort, then, comes from creating the expected UI pieces and then hooking them up to the TanStack model, along with the various APIs we need to design for the table component itself.
+### Globalization
 
-### Virtualization
+The order of table columns is inverted in RTL layouts.
 
-In order for the Nimble table to support numbers of rows in the thousands or higher, it's imperative that the DOM representation of the rows is virtualized, meaning only the necessary elements are in the DOM at any given moment.
+_**Note**:_ This is flagged as "out of scope" for the initial release.
 
-TanStack offers another library that helps out with this in [TanStack Virtual](https://tanstack.com/virtual/v3). This library essentially provides a means for supplying a set of state (i.e., the scrollable element, number of total rows, size estimate for row, etc...), and returns a set of "virtual items" that provide the state needed to render the appropriate views. Many of our prototype branches demonstate how we leveraged this, such as [`tanstack-virtualized-nimble-table`](https://github.com/ni/nimble/tree/tanstack-virutalized-nimble-table) ([Storybook](https://60e89457a987cf003efc0a5b-haosfwmjoq.chromatic.com/iframe.html?args=&id=table--table-story&viewMode=story)).
+### Performance
 
-## Alternative Implementations / Designs
+We intend for the `nimble-table` to handle both the rendering and interactive operations, such as sorting, in a near-instantaneous fashion on datasets of at least 10K rows. Between 10K and 100K rows of data on the client-side, however, we expect to see a notable drop in performance.
 
-### Perspective
+Clients, at least initially, will be expected to create mechanisms to restrict the amount of data coming to the client-side.
```

Comment:
"at least initially"? I don't understand the implication here. The tanstack-based nimble table has no direct plan for  >> 50K elements right? That would require a biggere architecture change (async compute in workers, etc).

Reply:
Re-worded this block to "clients will be expected to limit the amount of data being sent to the table to ideally a maximum of 10K rows".